src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -619,18 +619,17 @@ static Option captureCallState(String... capturedState) {
 
          /**
          * {@return A struct layout that represents the layout of the capture state segment that is passed
-         *          to a downcall handle linked with {@link #captureCallState(String...)}}
+         *          to a downcall handle linked with {@link #captureCallState(String...)}}.
          * <p>
-         * The capture state layout is <em>platform dependent</em>.
-         * The following example demonstrates how to print the element names of
-         * the current capture state layout on the console:
-         * {@snippet lang = "java":
-         *     captureStateLayout().memberLayouts().stream()
-         *         .map(MemoryLayout::name)
-         *         .filter(Optional::isPresent)
-         *         .map(Optional::get)
-         *         .forEach(System.out::println);
-         * }
+         * The capture state layout is <em>platform dependent</em> but is guaranteed to be
+         * a {@linkplain StructLayout struct layout} containing only {@linkplain ValueLayout value layouts}
+         * and possibly {@linkplain PaddingLayout padding layouts}.
+         * As an example, on Windows, the returned layout might contain three value layouts named:
+         * <ul>
```

Comment:
Almost there - instead of listing the names, just put a code snippet with the struct layout! That will make it crystal clear.